### PR TITLE
Remove deprecated properties

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
- These were scheduled to be removed with released 2.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,27 @@ Changelog
 ### Changed
 
 ### Removed
+* Deprecated `vlan` private-vlan properties:
```
Comment (mikewiebe) [inline CHANGELOG.md]: Can you add a note to indicate which `node_utils` objects should be used for `pvlan` capabilities?

**Comment by mikewiebe** [comment]
Are there any changed required on the puppet side?

File: CHANGELOG.md (hunk 1 above)
Comment (chrisvanheuveln) [reply CHANGELOG.md]: done

**Comment by chrisvanheuveln** [comment]
> Are there any changed required on the puppet side?

Yes. 
https://github.com/cisco/cisco-network-puppet-module/pull/557

**Comment by mikewiebe** [comment]
👍

**Pull request merged**